couchbase-persistence/couchbase-persistence-scala-sbt/hello-impl/src/test/scala/org/example/hello/impl/HelloEntitySpec.scala

```diff
@@ -0,0 +1,46 @@
+package org.example.hello.impl
+
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import com.lightbend.lagom.scaladsl.testkit.PersistentEntityTestDriver
+import com.lightbend.lagom.scaladsl.playjson.JsonSerializerRegistry
+import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpec}
+
+class HelloEntitySpec extends WordSpec with Matchers with BeforeAndAfterAll {
+
+  private val system = ActorSystem(
+    "HelloEntitySpec",
+    JsonSerializerRegistry.actorSystemSetupFor(HelloSerializerRegistry)
+  )
+
+  override protected def afterAll(): Unit = {
+    TestKit.shutdownActorSystem(system)
+  }
+
+  private def withTestDriver(
+    block: PersistentEntityTestDriver[HelloCommand[_],
+                                      HelloEvent,
+                                      HelloState] => Unit
+  ): Unit = {
+    val driver =
+      new PersistentEntityTestDriver(system, new HelloEntity, "hello-1")
+    block(driver)
+    driver.getAllIssues should have size 0
+  }
+
+  "hello entity" should {
+
+    "say hello by default" in withTestDriver { driver =>
+      val outcome = driver.run(Hello("Alice"))
+      outcome.replies should contain only "Hello, Alice!"
+    }
+
+    "allow updating the greeting message" in withTestDriver { driver =>
+      val outcome1 = driver.run(UseGreetingMessage("Hi"))
+      outcome1.events should contain only GreetingMessageChanged("Alice", "Hi")
```

Comment:
This assertion fails. `Alice` is not the  `id` of the entity instance, it's `hello-1` as set up in line `26`. The contents and this assertion are customized in this recipe (not part of the g8 template) which is why this test passes by default and not here.
```suggestion
      outcome1.events should contain only GreetingMessageChanged("hello-1", "Hi")
```